src/components/userProfileLayout/userProfileHeader/userProfileHeader.tsx

```diff
@@ -1,5 +1,62 @@
-function UserProfileHeader1() {
-	return <div>UserProfileHeader1</div>;
-}
+"use client";
 
-export default UserProfileHeader1;
+import { useEffect, useState } from "react";
+
+
+import Image from "next/image";
+import Link from "next/link";
+import cn from "classnames";
+
+import { ICurrentRate } from "../../../types/common/ComponentsProps";
+import cryptoIcon from "../../../assets/components/userProfile/crypto.svg";
+import cryptoBlackIcon from "../../../assets/components/userProfile/cryptoBlack.svg";
+import infoIcon from "../../../assets/components/userProfile/infoIcon.svg";
+
+import style from "./userProfileHeader.module.scss";
+
+const UserProfileHeader = ({ dollar, euro, crypto }: ICurrentRate) => {
+	const [currentDate, setCurrentDate] = useState<string>("");
+	const LAST_TWO_DIGITS = -2;
```

Comment:
better move this constant to separate file in helpers and use here nad in other places